mqtt-connect/websocket/src/main/java/com/cumulocity/mqtt/connect/client/MqttSubscriber.java

```diff
@@ -0,0 +1,22 @@
+package com.cumulocity.mqtt.connect.client;
+
+/**
+ * Interface for subscribing to messages from the server.
+ * <p>
+ * {@link MqttSubscriber} instances are setup using {@link MqttClient#buildSubscriber(MqttConfig)}
```

Comment:
suggestion: As commented for MqttPublisher, would it be better to say '... instances are created using ...'?

Reply:
Done too.